classes/Category.php

```diff
@@ -652,7 +652,6 @@ public static function getAllCategoriesName(
 				'.(isset($groups) && Group::isFeatureActive() ? 'LEFT JOIN `'._DB_PREFIX_.'category_group` cg ON c.`id_category` = cg.`id_category`' : '').'
 				'.(isset($idRootCategory) ? 'RIGHT JOIN `'._DB_PREFIX_.'category` c2 ON c2.`id_category` = '.(int) $idRootCategory.' AND c.`nleft` >= c2.`nleft` AND c.`nright` <= c2.`nright`' : '').'
 				WHERE 1 '.$sqlFilter.' '.($idLang ? 'AND `id_lang` = '.(int) $idLang : '').'
-				'.($active ? ' AND c.`active` = 1' : '').'
```

Comment:
@aleeks I don't need disbles ones, I need all the categories (enbled and disabled), should I create new function?